telegram/bot.py

```diff
@@ -3348,18 +3369,32 @@ def send_poll(self,
                   chat_id,
                   question,
                   options,
+                  is_anonymous=True,
+                  type=Poll.REGULAR,
+                  allows_multiple_answers=False,
+                  correct_option_id=None,
+                  is_closed=None,
                   disable_notification=None,
                   reply_to_message_id=None,
                   reply_markup=None,
                   timeout=None,
                   **kwargs):
         """
-        Use this method to send a native poll. A native poll can't be sent to a private chat.
+        Use this method to send a native poll.
 
         Args:
             chat_id (:obj:`int` | :obj:`str`): Unique identifier for the target private chat.
             question (:obj:`str`): Poll question, 1-255 characters.
             options (List[:obj:`str`]): List of answer options, 2-10 strings 1-100 characters each.
+            is_anonymous (:obj:`bool`, optional): True, if the poll needs to be anonymous,
+                defaults to True.
+            type (:obj:`str`, optional): Poll type, “quiz” or “regular”, defaults to “regular”.
```

Comment:
Maybe link to `Poll.QUIZ/REGULAR` here as well,  if you do that in the `Poll` docstring

Reply:
good catch, ty